spec.bs

```diff
@@ -2807,21 +2808,19 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=list=] of [=bid debug report
   1. Let |ad| be the [=interest group ad=] from |winningGroup|'s
      [=interest group/ad components=] where the [=interest group ad/render url=]
      equals |componentAd|. If there is no matching element, return failure.
-  1. [=list/Append=] |ad| to |winningAdComponents|.
+  1. [=list/Append=] |ad| onto |winningAdComponents|.
+1. Let |bidValue| be a new [=bid with currency=] with the following [=struct/items=]:
+  : [=bid with currency/value=]
+  :: |response|'s [=server auction response/bid=]
+  : [=bid with currency/currency=]
+  :: |response|'s [=server auction response/bid currency=]
 1. Let |winningBid| be a new [=generated bid=] with the following [=struct/items=]:
-   : [=generated bid/id=]
-   :: (0, 0)
-   : [=generated bid/for k-anon auction=]
-   :: TODO
-   : [=generated bid/bid=]
-   :: A new [=bid with currency=] with the following [=struct/items=]:
-      : [=bid with currency/value=]
-      :: |response|'s [=server auction response/bid=]
-      : [=bid with currency/currency=]
-      :: |response|'s [=server auction response/bid currency=]
-
+  : [=generated bid/id=]
+  :: (0, 0)
+  : [=generated bid/bid=]
+  :: |bidValue|
   : [=generated bid/bid in seller currency=]
-  :: TODO
+  :: Null
   : [=generated bid/ad=]
   :: |response|'s [=server auction response/ad render url=]

```

Comment:
[[=generated bid/ad=]](https://wicg.github.io/turtledove/#generated-bid-ad) is actually the [ad metadata](https://crsrc.org/c/content/browser/interest_group/interest_group_auction.h;drc=451093bbaf7fe812bf67d27d760f3bb64c92830b;l=345) ([usage in spec](https://github.com/WICG/turtledove/blob/main/spec.bs#L2123))  (sorry about the confusing names, should have been called it ad metadata in generated bid, which I'd like to fix when I get time).

ad render url should be part of the next field [[=generated bid/ad descriptor=]](https://wicg.github.io/turtledove/#ad-descriptor), which has render url and size.